afqbrowser/browser.py

```diff
@@ -11,8 +11,8 @@
     from http.server import SimpleHTTPRequestHandler
     import socketserver
 except ImportError:
-    from SimpleHTTPServer import SimpleHTTPRequestHandler
-    import SocketServer as socketserver
+    from http.server import SimpleHTTPRequestHandler
```

Comment:
This is here for Python2/Python3 compatibility. Note lines 11-12 above do the Python3 import.